### PR TITLE
Update apachevhost configs

### PR DESCRIPTION
**Pull request by mayankkapoor**
Now the configs will match what is present in production.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -28,9 +28,10 @@
  $install_dir = '/var/lib',
  $username    = 'pkgbuilder',
  $group       = 'www-data',
- $public_ip   = '127.0.0.1',
+ $public_ip   = '*',
```
Comment (anshprat) [inline manifests/init.pp]: public_ip is used elsewhere as well, so cannot change it to *.

./templates/settings.py.erb:104:BUILDSVC_REPOS_BASE_URL = 'http://<%= @public_ip %>/apt'
./templates/settings.py.erb:133:GITHUB_AUTH_CALLBACK = 'http://<%= @public_ip %>:<%= @public_port %>/'

**Comment by mayankkapoor** [comment]
@anshprat Good catch. Reverted that change. We'll keep manually changing this setting on tools01 for now, until we store this value in hiera.

File: manifests/apachevhost.pp [hunk 2]
```diff
@@ -8,8 +8,9 @@
  apache::vhost { 'pkgbuilder-vhost':
  port              => 80,
  docroot           => "${pkgbuilder::install_dir}/pkgbuilder",
- servername        => "localhost",
+ servername        => "${pkgbuilder::server_name}",
  vhost_name        => "${pkgbuilder::public_ip}",
```
Comment (anshprat) [inline manifests/apachevhost.pp]: vhost_name        => "${pkgbuilder::public_ip}",

Shall we change this to '*' instead? Anyways this module is not generic - so lets just get it working? Basically making any further manual changes makes me squirm.

Comment (anshprat) [inline manifests/apachevhost.pp]: Or make it proper by adding a variable to class definition and then use that.

**Comment by mayankkapoor** [comment]
@anshprat ready for review.

**Comment by anshprat** [comment]
Looks good, merging

**Pull request merged**